server/src/Scripting/v8/V8PlayerImpl.cpp

```diff
@@ -1241,6 +1241,9 @@ void Player_Function_Join(const v8::FunctionCallbackInfo<v8::Value>& args)
 	}
 }
 
+// TODO (samich): Consider this deprecated based on the behavior of PLO_TRIGGERACTION.
+//                Since "clientside" is the only action the client responds to, we can
+//                rely on Player_Function_TriggerClient instead.
 void Player_Function_TriggerAction(const v8::FunctionCallbackInfo<v8::Value>& args)
 {
```

Comment:
may be possible to send a triggeraction to a player that triggers a clientside npc, ex:

player.triggeraction(player.x + 1.5, player.y, "blowup", 3);

//#CLIENTSIDE
function onActionBlowUp(params) {
  sleep(params[0]);
  doExplosion();
}

or something similar